confluent_server/bin/confluent_selfcheck

```diff
@@ -237,6 +237,7 @@ if __name__ == '__main__':
         allok = True
         uuidok = False
         macok = False
+        valid_nodes = [node for node in sess.read('/nodes/')] #get all valiv nodes
         for rsp in sess.read(f'/nodes/{args.node}/attributes/all'):
```

Comment:
This would not work, the right list comprehension would be:
[node['item']['href'][:-1] for node in c.read('/nodes/')]